src/query/ast/src/ast/statements/table.rs

```diff
@@ -698,13 +698,17 @@ pub enum CompactTarget {
 #[derive(Debug, Clone, PartialEq, Drive, DriveMut)]
 pub struct VacuumTableOption {
     #[drive(skip)]
-    pub dry_run: bool,
+    // Some(true) means summary
+    pub dry_run: Option<bool>,
```

Comment:
Seems we need a new filed: `pub dry_run_summary: bool`

Reply:
The syntax is easy to understand:
`VACUUM ... [DRY RUN] | [DRY RUN SUMMARY]`

than:
`VACUUM ... [DRY RUN] [SUMMARY]`

Reply:
if we have `dry_run_summary` option, it means that is option is independent with `dry_run` option, but it is not an independent.so:

* dry_run.is_none(): not dry run
*  dry_run.is_some(summary): dry run with summary option.

Reply:
OK.
After the explain it's make sense now, better  to add the above comments to the code to make it more understandable.